lib/Core/Site/Values/Content.php

```diff
@@ -388,11 +395,17 @@ private function getOwner()
             return $this->owner;
         }
 
-        try {
-            $this->owner = $this->site->getLoadService()->loadContent($this->getContentInfo()->ownerId);
-        } catch (NotFoundException $e) {
-            // Do nothing
-        }
+        $loadService = $this->site->getLoadService();
+        $ownerId = $this->getContentInfo()->ownerId;
+        $this->owner = $this->repository->sudo(
+            function() use ($loadService, $ownerId) {
+                try {
+                    return $loadService->loadContent($ownerId);
+                } catch (NotFoundException $e) {
+
```

Comment:
Please add a comment (`// Do nothing`) indicating that the empty catch is intentional